app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java

```diff
@@ -26,6 +27,9 @@
     @Inject
     ClearCache clearCache;
 
+    @Inject
+    NotificationStrategy notificationStrategy;
```

Comment:
I think it's not good to increase Strategy classes to Application class.
I'll create Provider class for debot for the future.